frontend/src/index.js

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { App } from './components';
+import { App } from './components/index';
```

Comment:
You don't need index at the end of imports like this. There's a module used by CRA that takes care of resolving it automatically.